docker/build/installers/install_python_modules.sh

```diff
@@ -29,8 +29,7 @@ apt update -y && apt install -y \
     python3-pip \
     python3-psutil \
     python3-scipy \
-    software-properties-common \
-    libasound2-dev
```

Comment:
So, libasound is no longer required ?

Reply:
libasound is installed as a dependency of pyalsaaudio however, I haven't found where the pyalsaaudio is used